hudi-spark-datasource/hudi-spark/src/test/java/org/apache/hudi/table/action/compact/TestInlineCompaction.java

```diff
@@ -204,12 +205,12 @@ public void testSuccessfulCompactionBasedOnNumOrTime() throws Exception {
     // Given: make three commits
     HoodieWriteConfig cfg = getConfigForInlineCompaction(3, 60, CompactionTriggerStrategy.NUM_OR_TIME);
     try (SparkRDDWriteClient<?> writeClient = getHoodieWriteClient(cfg)) {
-      List<HoodieRecord> records = dataGen.generateInserts(writeClient.createNewInstantTime(), 10);
+      List<HoodieRecord> records = dataGen.generateInserts(WriteClientTestUtils.createNewInstantTime(), 10);
```

Comment:
The instant time can be possibly removed from the APIs of test generator as the instant time is solely used for constructing String values.  Something for a later refactoring.